keras/src/export/lite_rt_exporter.py

```diff
@@ -0,0 +1,452 @@
+import tensorflow as tf
+from keras.src.export.export_utils import get_input_signature, make_tf_tensor_spec
+import tempfile
+import os
+import numpy as np
+
+# Try to import LiteRT AOT compilation if available
+try:
+    from litert.python.aot import aot_compile
+    from litert.python.aot.core import types as litert_types
+    from litert.python.aot.vendors import import_vendor
+    LITERT_AVAILABLE = True
+except ImportError:
+    LITERT_AVAILABLE = False
+
+
+class LiteRTExporter:
+    """
+    Exporter for the LiteRT (TFLite) format that creates a single,
+    callable signature for `model.call`.
+    """
+
+    def __init__(self, model, input_signature=None, verbose=0, 
+                 aot_compile_targets=None, **kwargs):
+        """Initialize the LiteRT exporter.
+        
+        Args:
+            model: The Keras model to export
+            input_signature: Input signature specification
+            verbose: Verbosity level (0=quiet, 1=info)
+            aot_compile_targets: List of LiteRT targets for AOT compilation
+            **kwargs: Additional export parameters
+        """
+        self.model = model
+        self.input_signature = input_signature
+        self.verbose = verbose
+        self.aot_compile_targets = aot_compile_targets
+        self.kwargs = kwargs
+
+    def export(self, filepath):
+        """Exports the Keras model to a TFLite file and optionally performs AOT compilation.
+        
+        Args:
+            filepath: Output path for the exported model
+            
+        Returns:
+            Path to exported model or compiled models if AOT compilation is performed
+        """
+        if self.verbose:
+            print("Starting LiteRT export...")
+
+        # 1. Ensure the model is built by calling it if necessary
+        self._ensure_model_built()
+
+        # 2. Resolve / infer input signature
+        if self.input_signature is None:
+            if self.verbose:
+                print("Inferring input signature from model.")
+            self.input_signature = get_input_signature(self.model)
+        
+        # 3. Convert the model to TFLite.
+        tflite_model = self._convert_to_tflite(self.input_signature)
+        
+        if self.verbose:
+            final_size_mb = len(tflite_model) / (1024*1024)
+            print(f"TFLite model converted successfully. Size: {final_size_mb:.2f} MB")
+        
+        # 4. Save the initial TFLite model to the specified file path.
+        if not filepath.endswith('.tflite'):
```

Comment:
Again we want consistency with other exporters. We error for the wrong file extension elsewhere, so we should do the same here. https://github.com/keras-team/keras/blob/cdc7e2993e173325cd99132e07b9dd9e0ea84aa3/keras/src/export/openvino.py#L52-L55